jep/223/README.adoc

```diff
@@ -86,8 +86,44 @@ The changes to the existing permissions are detailed in the diagram below.
 
 image::Targetted_permission_model.png[Design of the new permission model.]
 
+Specifically, for an installation of Jenkins v2.207 with the default suggested plugins, the following configuration items would be available to a user with the `Overall/Manage` permission:
+
+* `<JENKINS_URL>/manage`
+** `Configure System`
+** `Reload Configuration From Disk`
+** `System Information`
+** `Load Statistics`
+** `Jenkins CLI`
+** `About Jenkins`
+** `Manage Old Data`
```

Comment:
Still disagree with this, same feedback as I previously provided out of band: If Manage doesn't imply global Job/Read, this will provide information about jobs and builds, and some parts of their configuration.

Reply:
right, i am looking at that this morning and will probably remove this